test/Microsoft.AspNet.Routing.Tests/Template/TemplateMatcherTests.cs

```diff
@@ -923,7 +924,7 @@ private TemplateMatcher CreateMatcher(string template, object defaults = null)
                 defaults ?? new Dictionary<string, object>());
 
             // Act
-            var match = matcher.Match(path);
+            var match = matcher.Match(new PathString(path));
```

Comment:
This isn't really needed, but it makes things more explicit for someone else that has to debug this in the future.